src/Utils/MockBoot.tsx

```diff
@@ -0,0 +1,21 @@
+import { Breakpoint } from "@artsy/palette/dist/Theme"
```

Comment:
Can import this from the root as all exports are now available there. 

`import { Breakpoint } from '@artsy/palette'`